apps/web/components/OrderPropertyFileUpload/OrderPropertyFileUpload.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div class="w-full">
+    <label :for="`prop-${productPropertyGetters.getOrderPropertyId(productProperty)}`">
+      {{ $t('orderProperties.upload.orderPropertyTypeFile') }}
```

Comment:
Shouldn't we display the order property name with the addon: '- Type: File' ?

Reply:
hm, can you give an example?
"-Type:File should also be translated"

Reply:
`{{ productPropertyGetters.getOrderPropertyName(productProperty) }} - {{ $t('orderProperties.upload.orderPropertyTypeFile') }}`
And `orderProperties.upload.orderPropertyTypeFile` should be:
**EN:** Type: File
**DE:** Typ: Datei